### PR TITLE
updating Toolkit icon and missing apps in root readme

### PR DESCRIPTION
**Pull request by Jegadeesh-MSFT**

### CODE REVIEW

**Review by Prithvi-MSFT — APPROVED** [review]
changes look fine

File: README.md [hunk 1]
```diff
@@ -159,7 +161,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 
 |    | Sample Name    | Description                                                                      | C#    | JavaScript      |App Manifest|
 |:--:|:---------------|:---------------------------------------------------------------------------------|:------|:----------------|:-------|
-|1|Task Modules          | Sample app showing off the Teams Task Module, a way to invoke custom code from a bot, a tab, or both! |[View][bot-task-module#cs]     |[View][app-task-module#ts]|[View](/samples/bot-task-module/csharp/demo-manifest/bot-task-module.zip)
+|1|Task Modules          | Sample app showing off the Teams Task Module, a way to invoke custom code from a bot, a tab, or both! |[View][app-task-module#cs]     |[View][app-task-module#ts]|[View](/samples/bot-task-module/csharp/demo-manifest/bot-task-module.zip)
```
Comment (Pawank-MSFT) [inline README.md:164]: this should be bot-task-module.

Comment (Jegadeesh-MSFT) [reply README.md]: Fixed

**Comment by Jegadeesh-MSFT** [comment]
**Executed pipeline**

image
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/6347637531

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**